### ISSUE TITLE
L063 example isn't relevant

### ISSUE THREAD

**Issue by jivanpal** [sqlfluff/sqlfluff#3309]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### Links or command line

- L063: https://docs.sqlfluff.com/en/stable/rules.html#sqlfluff.core.rules.Rule_L063
- L040: https://docs.sqlfluff.com/en/stable/rules.html#sqlfluff.core.rules.Rule_L040

### Issue/Suggested Improvement

Looks like L063's example is identical to that of L040, and doesn't reflect what the rule actually does. Here's a suggested example instead:

```sql
CREATE TABLE t (
    a int unsigned,
    b VARCHAR(15)
);
```

Which SQLFluff could correct to:

```sql
CREATE TABLE t (
    a INT UNSIGNED,
    b VARCHAR(15)
);
```

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
